src/index.ts

```diff
@@ -10,12 +10,14 @@ setupDataDog()
 // tslint:disable:no-console
 import "reflect-metadata"
 
+import * as Sentry from "@sentry/node"
 import { GraphQLServer, Options } from "graphql-yoga"
 import { parse } from "mongodb-uri"
 import * as morgan from "morgan"
 import { Connection, createConnection } from "typeorm"
 import { databaseConfig } from "./config/database"
 import { createSchema } from "./utils/createSchema"
+const enableSentry = !!SENTRY_PRIVATE_DSN
```

Comment:
what is this `!!` syntax?

Reply:
it's the same as `Boolean(SENTRY_PRIVATE_DSN)` it basically converts an truthy or falsy value to a proper boolean

Reply:
`Boolean(SENTRY_PRIVATE_DSN)` might actually be better.

Reply:
Agreed that seems more communicative.